src/components/Header/Header.jsx

```diff
@@ -114,9 +114,9 @@ class Header extends React.Component {
       children: <ActionSettings />,
     };
 
-    const LogoGroup = ({ small }) => (
+    const LogoGroup = ({ logoSmall = small }) => (
```

Comment:
remove the default value, also this may be a breaking change if the code using this component isn't updated

Reply:
Reverting this change for now in case of breaks, until fixed it is in violation of the `no-shadow` rule (https://eslint.org/docs/rules/no-shadow).